### PR TITLE
Registration email#66

### LINKED ISSUE #66: User Story 9, Registration Email must be unique
```
As a visitor
When I visit the user registration page
If I fill out the registration form
But include an email address already in the system
Then I am returned to the registration page
My details are not saved and I am not logged in
The form is filled in with all previous data except the email field and password fields
I see a flash message telling me the email address is already in use
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #69

depends on #68

### PR DESCRIPTION
**Pull request by danbriechle**
Creates sad path test as per user story 9
Adds logic to user create method for flash messages per uniqueness of email
Adds to user new.html.erb to display messages and auto complete form 
Tests passing

will and dan driver / navigator
closes #66

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@ def create
       redirect_to profile_path
       flash[:success] = "You are now registered and logged in."
     else
+        if @user.errors.details[:email] != nil
```
Comment (asmolentzov) [inline app/controllers/users_controller.rb]: Shouldn't need `!=nil` since will check for truthy
